src/algebra/add_torsor.lean

```diff
@@ -135,6 +135,12 @@ end
   g +ᵥ p₁ = g +ᵥ p₂ ↔ p₁ = p₂ :=
 ⟨vadd_left_cancel g, λ h, h ▸ rfl⟩
 
+variables (P)
+
+/-- Adding the group element `g` to a point is an injective function. -/
+lemma vadd_left_injective (g : G) : function.injective (λ p : P, g +ᵥ p) :=
```

Comment:
You could also state this lemma as follows, to match `vadd_right_injective` (and the same holds for `vsub_right_injective`). I have no preferences for either `λ`'s or operator sections, except that I would prefer consistency.
```suggestion
lemma vadd_left_injective (g : G) : function.injective ((+ᵥ) g) :=
```